app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
Same comment applies here -- because we're showing who left a comment, either we need to delete a user's comments when the account is deleted, or have some way of showing that the user no longer exists